content/_layouts/developerguide.html.haml

```diff
@@ -0,0 +1,5 @@
+---
+layout: developer
```

Comment:
If you're confident they're going to differ in the future then that's fine. This isn't Java so redundant "hierarchies" aren't necessary :smiling_imp: